list.js

```diff
@@ -9,45 +9,60 @@ const liNote = document.querySelector(".note");
 // Buttons
 const btnMainAddNewTask = document.querySelector(".btn-add");
 const btnAddToList = document.querySelector(".btn-add-form");
-const btnCancel = document.querySelector(".btn-cancel");
-const btnMore = document.querySelector(".btn-more");
-
-// Array
-const listArr = [];
-let id = 0;
 
 // Functions
 const clearInput = function () {
   taskNameInput.value = "";
   taskNoteInput.value = "";
 };
 
-// add TASK to the listArr so later you can remove it
-const addListArr = function (toDo, note, id, done, trash) {
-  listArr.push({
-    name: toDo,
-    note: note,
-    id: id,
-    done: false,
-    trash: false,
+const createListEl = function (toDo, note) {
+  // Creating new elements
+  const newLi = document.createElement("li");
+  const h3Text = document.createElement("span");
+  const newNote = document.createElement("p");
+  const btnTrash = document.createElement("button");
+  const btnMore = document.createElement("button");
+
+  // Adding text to elements
+  h3Text.textContent = `${toDo}`;
+  newNote.textContent = `${note}`;
+  btnTrash.textContent = `delete`;
+  btnMore.textContent = `more_horiz`;
+
+  // Adding classes to elements
+  newLi.classList.add("task");
+  newNote.classList.add("note", "hidden");
+  btnTrash.classList.add(
+    "btn",
+    "btn-form",
+    "btn-trash",
+    "material-symbols-outlined"
+  );
+  btnMore.classList.add(
+    "btn",
+    "btn-form",
+    "btn-more",
+    "material-symbols-outlined"
+  );
+
+  // Adding event listeners to buttons
+  btnTrash.addEventListener("click", () => {
+    if (confirm("Your task will be deleted")) {
+      newLi.remove();
+    }
   });
-  id++;
-};
 
-// add DONE class to checkbox, TRASH to
-const createList = function (toDo, note, id, done, trash) {
-  if (taskNameInput.value) {
-    const text = `
-    <li class="task" id=${id}>
-    <span class="to-do">${toDo}</span>
-    <button class="btn btn-form btn-cancel">cancel</button>
-    <button class="btn btn-form btn-more">more</button>
-    <br />
-    <span class="note hidden">Note: ${note}</span>
-  </li>`;
-    unorderedList.insertAdjacentHTML("beforeend", text);
-    clearInput();
-  } else alert("add task name");
+  btnMore.addEventListener("click", () => newNote.classList.toggle("hidden"));
+
+  // Appending children
+  newLi.appendChild(h3Text);
+  newLi.appendChild(btnTrash);
+  newLi.appendChild(btnMore);
+  newLi.appendChild(newNote);
+  unorderedList.appendChild(newLi);
+
+  console.log(newLi);
```

Comment:
Is that needed? If you are using console log to debug your code - remove it before merge

Reply:
I used it to debug and I must've overlooked it as I was reviewing my code.

Reply:
Yeah - I totally understand - you needed them. However, remember when creating code in. the future - it's one of the gold rules of clean code :) Descriptive variables names